src/popup/router/components/AccountInfo.vue

```diff
@@ -265,7 +261,7 @@ export default {
       .claim-chainname {
         @extend %face-sans-14-medium;
 
-        line-height: 16px;
+        line-height: 21px;
```

Comment:
please check this recommendation https://github.com/aeternity/superhero-wallet/pull/883#discussion_r594110313

Reply:
We'll rethink this on additional account card rebranding. I've created an issue with a link to this comment We'llhttps://github.com/aeternity/superhero-wallet/issues/1049